CheatSheet.md

```diff
@@ -273,16 +271,18 @@ can be written as follows:
 
 Pattern matches are also used quite often in anonymous functions:
 ```scala
-    val pairs: List[(Char, Int)] = ('a', 2) :: ('b', 3) :: Nil
+    val pairs: List[Option[Char]] = Some('a') :: None :: Some('b') :: Nil
```

Comment:
This is not a list of pairs anymore

Reply:
Good catch! I thought that `Option` was better to show anonymous pattern match, as it is not required anymore for tuples

Reply:
Oh, I understand now. Maybe just rename `pairs` to `options`?